yarn.lock

```diff
@@ -10345,7 +10345,7 @@ qs@6.9.7:
 
 qs@~6.5.2:
   version "6.5.3"
-  resolved "https://registry.yarnpkg.com/qs/-/qs-6.5.3.tgz#3aeeffc91967ef6e35c0e488ef46fb296ab76aad"
+  resolved "https://artifacts.aue1e.internal:443/artifactory/api/npm/npm-okta-master/qs/-/qs-6.5.3.tgz#3aeeffc91967ef6e35c0e488ef46fb296ab76aad"
```

Comment:
fix to an Artifactory URL not accessible externally

Reply:
That's strange I can download file from `https://registry.yarnpkg.com/qs/-/qs-6.5.3.tgz#3aeeffc91967ef6e35c0e488ef46fb296ab76aad` it should be available.
If we change dependency to artifactory that will break the build for everyone outside of Okta VPN.

Reply:
oh this is odd; no the intent was to REMOVE the artifactory URL and REPLACE with the yarnpkg.com one.  maybe something else is up.... looking....

Reply:
yeah very odd. every time i run yarn locally, it reverts to the artifactory link. i'd guess it's doing this in CI also? wonder if it does it on your local. regardless, reverted it to keep whats in the repo...